### PR TITLE
Add SuchtCraft

### PR DESCRIPTION
**Pull request by WylderPanda**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/suchtcraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "suchtcraft",
+  "nice_name": "SuchtCraft",
+  "direct_ip": "SuchtCraft.net",
+  "social": {
+    "web": "https://suchtcraft.net/",
+    "web_support": "https://suchtcraft.net/forum/",
+    "discord": "https://suchtcraft.net/discord",
```
Comment (JakobBruening) [inline minecraft_servers/suchtcraft/manifest.json:8]: "Own urls" are only supported for partners. Please use the official discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

File: minecraft_servers/suchtcraft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "suchtcraft",
+  "nice_name": "SuchtCraft",
+  "direct_ip": "SuchtCraft.net",
+  "social": {
+    "web": "https://suchtcraft.net/",
+    "web_support": "https://suchtcraft.net/forum/",
```
Comment (JakobBruening) [inline minecraft_servers/suchtcraft/manifest.json:7]: Remove "/" at the end.

File: minecraft_servers/suchtcraft/manifest.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      "color": "#5454FC"
+    },
+	"challenges": {
+      "name": "Challenges",
+      "color": "#5454FC"
+    }
+  },
+  "discord": {
+    "server_id": 538780470192766986,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/suchtcraft/manifest.json:24]: Please set to "false", as this is the default value.

File: minecraft_servers/suchtcraft/manifest.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "discord": "https://suchtcraft.net/discord",
+    "twitter": "SuchtCraftNET",
+    "instagram": "suchtcraftnet"
+  },
+  "gamemodes": {
+    "stadt": {
+      "name": "Stadt",
+      "color": "#5454FC"
+    },
+	"challenges": {
```
Comment (JakobBruening) [inline minecraft_servers/suchtcraft/manifest.json:17]: Please fix formatting here.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/suchtcraft/manifest.json (hunk 1 above)
Comment (WylderPanda) [reply minecraft_servers/suchtcraft/manifest.json]: Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a

File: minecraft_servers/suchtcraft/manifest.json (hunk 2 above)
Comment (WylderPanda) [reply minecraft_servers/suchtcraft/manifest.json]: Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a

File: minecraft_servers/suchtcraft/manifest.json (hunk 3 above)
Comment (WylderPanda) [reply minecraft_servers/suchtcraft/manifest.json]: Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a

File: minecraft_servers/suchtcraft/manifest.json (hunk 4 above)
Comment (WylderPanda) [reply minecraft_servers/suchtcraft/manifest.json]: Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a

**Review by JakobBruening — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**